Makefile

```diff
@@ -153,18 +153,16 @@ buildconfig:
 	git checkout -- kolibri/utils/build_config # restore __init__.py
 	python build_tools/customize_build.py
 
-dist: writeversion staticdeps staticdeps-cext buildconfig assets translation-django-compilemessages
+dist: writeversion staticdeps staticdeps-cext buildconfig translation-extract assets translation-django-compilemessages
 	python setup.py sdist --format=gztar --static > /dev/null # silence the sdist output! Too noisy!
 	python setup.py bdist_wheel --static
 	ls -l dist
 
 pex: writeversion
 	ls dist/*.whl | while read whlfile; do pex $$whlfile --disable-cache -o dist/kolibri-`cat kolibri/VERSION | sed 's/+/_/g'`.pex -m kolibri --python-shebang=/usr/bin/python; done
 
-translation-extract: clean-build
-	@echo ""
-	@echo "!! This assumes that you are running with latest assets. Run 'make assets' if in doubt !!"
-	@echo ""
+translation-extract:
+	yarn run makemessages
```

Comment:
Does this work without `make assets` or is that still a requirement? Or what's required to run, just `yarn install`?